connections/walletconnect.js

```diff
@@ -0,0 +1,128 @@
+const EventEmitter = require("events");
+const WC = require("@walletconnect/browser");
+const WCQRCode = require("@walletconnect/qrcode-modal");
+const dev = process.env.NODE_ENV === "development";
+
+let WalletConnect;
+
+class WalletConnectConnection extends EventEmitter {
+  constructor(_WalletConnect, url, options) {
+    super();
+
+    this.qrcode =
+      typeof options.qrcode === "undefined" || options.qrcode !== false;
```

Comment:
Nit: I personally don't care for chaining binaries like this and find something like

```
if (options.qrcode === "undefined || options.qrcode !== false) {
    this.qrcode = false;
} else {
    this.qrcode = true;
}
```

This may not be the right functionality as I haven't had my coffee LOL 😂

But in general I find `if/else` to be more readable than chaining booleans. This example isn't extreme so I'm probably splitting hairs here... but neverless, what do you think? Thanks! ☺️

Reply:
I generally think this just communicates author's intent a bit better. Thoughts? 👍

Reply:
I will replace this check by implementing a default qrcode value in options

Reply:
@floating thanks! Makes a lot of sense to me, this is a much more elegant approach that avoids the convo in the first place! 🔥